crates/zk-por-core/src/database.rs

```diff
@@ -16,6 +17,27 @@ impl UserId {
         rng.fill(&mut bytes);
         Self(bytes)
     }
+
+    pub fn to_string(&self) -> String {
+        self.0.encode_hex()
+    }
+
+    pub fn from_hex_string(hex_str: String) -> Self {
+        if hex_str.len() != 64 {
+            tracing::error!("User Id: {:?} is not a valid id, length is not 256 bits", hex_str);
```

Comment:
we need return error or exit program here

Reply:
Will take a look